src/__tests__/to-be-disabled.tsx

```diff
@@ -25,6 +25,7 @@ describe('.toBeDisabled', () => {
   Object.entries(ALLOWED_COMPONENTS).forEach(([name, Component]) => {
     test(`handle disabled prop for element ${name}`, () => {
       const { queryByTestId } = render(
+        // @ts-ignore how to type it?
```

Comment:
What is the problem here?

Reply:
The component type is not the good one and i'm not sure how to type it, if you want to give it a try feel free

Reply:
Could you paste the error here?

Reply:
Changed it